app/controllers/users_controller.rb

```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```

Comment:
- [ ] Dear @Caren-Koroeny you did a great job creating the user_controller, But we need to add the `#index` and `#show` actions. Please add those actions.